hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RandomKeyGenerator.java

```diff
@@ -484,8 +484,9 @@ private void printStats(PrintStream out) {
     out.println("Number of Volumes created: " + numberOfVolumesCreated);
     out.println("Number of Buckets created: " + numberOfBucketsCreated);
     out.println("Number of Keys added: " + numberOfKeysAdded);
-    out.println("Replication: " + replicationConfig.getReplication());
-    out.println("Replication type: " + replicationConfig.getReplicationType());
+    if (replicationConfig != null) {
+      out.println("Replication: " + replicationConfig);
+    }
```

Comment:
Do we want to remove ```out.println("Replication type: " + replicationConfig.getReplicationType());``` in the scope of fixing NPE?

I think we should just fix the NPE. In general changing CLI output is considered an incompatible change as well. May be if we want to remove it, we can remove it as part of a separate ticket

Reply:
I don't think the output of `ozone freon` is so important to guarantee any compatibility, especially the stats printed at exit.

Reply:
But still why to remove a line from the CLI which ain't creating any trouble and that too in scope of fixing a NPE.
anyway the change is in by the time I could circle back, should be fine then.....